controller/src/main/resources/reference.conf

```diff
@@ -13,5 +13,6 @@ dswarm {
     port = 8087
     context-path = "/dmp"
   }
+  reporting.elasticsearch.index=metrics
```

Comment:
Could you add this to persistence/src/main/resources/reference.conf at line 53 to the other settings and remove it from here?